tools/professor-prebid.md

```diff
@@ -97,6 +97,42 @@ Here's an example of an overlay:
 
 ![Overlay example](/assets/images/tools/professor-prebid-10.png)
 
+#### User Sync Network Inspector
+
+{: .alert.alert-info :}
+Note: This is a tool that does require some technical knowledge because it utilizes the Chrome Devtools.
+
+##### Use Case
+
+When SSP user sync URL’s are requested on a Publisher site, they typically will initiate calls or redirect to other resources which will initiate or redirect to other resources, etc.  When it comes to privacy compliance, it can be difficult to track down if all resources down the request chain are obeying all of the necessary privacy compliance rules.  Are all of the correct query parameters being passed where they should be down the chain?
+
+The goal of this tool doesn’t attempt to fix scenarios where compliance params might not be present, instead it is a tool designed to quickly show transparency around this. It is up to the SSP to be aware of how compliance params are being used and to act accordingly based on this.
+
+##### How to Use
+
+{: .alert.alert-info :}
+Note: It is advised when testing User Sync URL's that you clear cookies relative to the domain you are testing. This will ensure that results are in-line with an initial visit to the current page. Additionally, the first resource matching the root URL will be used to generate the initiator request chain.
+
+- With the Professor Prebid extension on, open the Chrome developer tools by right-clicking anywhere on a webpage and selecting `Inspect`.
```

Comment:
The markdown linter enforces the first list style operator in use. So in this document the first appearing list uses `*` , which causes the linter to complain on every usage of `-` . You may either change everything to `*` or `-` . As you like